image/image.go

```diff
@@ -65,7 +65,14 @@ func (i *genericImage) Manifest() ([]byte, string, error) {
 			return nil, "", err
 		}
 		i.cachedManifest = m
-		if mt == "" {
+		is_reqmt := false
+		for _, reqmt := range i.requestedManifestMIMETypes {
+			if mt == reqmt {
+				is_reqmt = true
+				break
+			}
+		}
+		if !is_reqmt {
```

Comment:
I agree to wait after skopeo copy and add that fix as a stopgap for now.

With a _great_ comment I think we can live with that fix for a while.

@mtrmac agree?

I couldn't figure out a better place for this detection now. I'll start reviewing the skopeo copy